pkgs/by-name/ga/gate/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+let
+  pname = "gate";
+  version = "0.36.7";
+in
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "minekube";
+    repo = "gate";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-WHxpx20O/HuCWqbY4zTxcjyIhW3+FQtTz5sUGAda71g=";
+  };
+
+  vendorHash = "sha256-dswNJQWqN+u/mnpbj9se2j9uEi0ewNTXVlN3WnNbcyg=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = {
+    description = "High-Performance, Low-Memory, Lightweight, Extensible Minecraft Reverse Proxy with Excellent Multi-Protocol Version Support - Velocity/Bungee Replacement - Ready for dev and large deploy";
```

Comment:
Minor-ish nit that slipped my mind, but I believe there is an implicit character limit for description. Try to restrict `description` to 80 characters, and add a longer description in `longDescription` instead